src/util/misc/objectTransforms.ts

```diff
@@ -76,12 +76,14 @@ export const applyTransformToObject = (
  * @param  {fabric.Object} target object to transform
  */
 export const resetObjectTransform = (target: FabricObject) => {
-  target.scaleX = 1;
-  target.scaleY = 1;
-  target.skewX = 0;
-  target.skewY = 0;
-  target.flipX = false;
-  target.flipY = false;
+  target.set({
+    flipX: false,
+    flipY: false,
+    scaleX: 1,
+    scaleY: 1,
+    skewX: 0,
+    skewY: 0,
+  });
```

Comment:
let's leave this as Object.assign(target, {});
today set has the particular meaning of `side effect needed` i do not want to offuscate which part of the code now are using it because necessary

Reply:
done